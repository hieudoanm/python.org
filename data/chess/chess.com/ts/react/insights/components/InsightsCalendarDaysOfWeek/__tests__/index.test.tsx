import { render } from '@testing-library/react';
import { Insights } from '@web/services/chess/chess.dto';
import { mockResizeObserver } from '@web/utils/mock-resize-observer';
import { InsightsCalendarDaysOfWeek } from '..';

describe('InsightsCalendarDaysOfWeek', () => {
  beforeEach(() => {
    mockResizeObserver();
  });

  it('to match snapshot', () => {
    const { container } = render(
      <InsightsCalendarDaysOfWeek insights={{} as Insights} />
    );
    expect(container).toMatchSnapshot();
  });
});
