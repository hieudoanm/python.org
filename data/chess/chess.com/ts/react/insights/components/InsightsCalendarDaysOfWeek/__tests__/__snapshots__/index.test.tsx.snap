// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InsightsCalendarDaysOfWeek to match snapshot 1`] = `
<div>
  <div
    class="card border-base-content border shadow"
  >
    <div
      class="border-base-content border-b px-8 py-4"
    >
      <div
        class="text-base md:text-lg lg:text-xl"
      >
        Day of Week
      </div>
    </div>
    <div
      class="card-body border-base-content border-b"
    >
      <div
        class="flex flex-col gap-y-8"
      >
        <p
          class="text-base font-bold md:text-lg"
        >
          Games by Days of Week
        </p>
      </div>
    </div>
    <div
      class="card-body border-base-content border-b"
    >
      <div
        class="flex flex-col gap-y-8"
      >
        <p
          class="text-base font-bold md:text-lg"
        >
          Accuracy by Days of Week
        </p>
      </div>
    </div>
    <div
      class="card-body border-base-content border-b"
    >
      <div
        class="flex flex-col gap-y-8"
      >
        <p
          class="text-base font-bold md:text-lg"
        >
          Results by Days of Week
        </p>
      </div>
    </div>
    <div
      class="card-body"
    />
  </div>
</div>
`;
